src/theory/bags/bags_rewriter.h

```diff
@@ -142,32 +142,32 @@ class BagsRewriter : public TheoryRewriter
 
   /**
    * rewrites for n include:
-   * - (difference_subtract A emptybag) = A
-   * - (difference_subtract emptybag A) = emptybag
-   * - (difference_subtract A A) = emptybag
+   * - (difference_subtract A bag.empty) = A
+   * - (difference_subtract bag.empty A) = bag.empty
+   * - (difference_subtract A A) = bag.empty
    * - (difference_subtract (union_disjoint A B) A) = B
    * - (difference_subtract (union_disjoint B A) A) = B
-   * - (difference_subtract A (union_disjoint A B)) = emptybag
-   * - (difference_subtract A (union_disjoint B A)) = emptybag
-   * - (difference_subtract A (union_max A B)) = emptybag
-   * - (difference_subtract A (union_max B A)) = emptybag
-   * - (difference_subtract (intersection_min A B) A) = emptybag
-   * - (difference_subtract (intersection_min B A) A) = emptybag
+   * - (difference_subtract A (union_disjoint A B)) = bag.empty
+   * - (difference_subtract A (union_disjoint B A)) = bag.empty
+   * - (difference_subtract A (union_max A B)) = bag.empty
+   * - (difference_subtract A (union_max B A)) = bag.empty
+   * - (difference_subtract (intersection_min A B) A) = bag.empty
+   * - (difference_subtract (intersection_min B A) A) = bag.empty
```

Comment:
Missing `bag.` prefix, and `intersection_min` -> `bag.inter_min`. There's more throughout the whole file, make sure to rename all of them.